### PR TITLE
created delete event test case, updated user search

### PR DESCRIPTION
**Pull request by mann-c**

### CODE REVIEW

**Review by sanjayalwani — APPROVED** [review]
Looks non-conflicting to me

**Review by sanjayalwani — CHANGES_REQUESTED** [review]
Wait a minute you change some fields in event join

File: index.js [hunk 1]
```diff
@@ -472,8 +472,8 @@ app.post('/event/join', (req,res) => {
               'JOIN',
               result.rows[0],                                                       //All event info
               {
-                login: req.user.data.login,                                         //Login of who joined
-                name: `${req.user.data.firstname} ${req.user.data.lastname}`        //Name of who joined
+                login: req.body.login,                                         //Login of who joined
+                name: `${req.body.firstname} ${req.body.lastname}`        //Name of who joined
```
Comment (sanjayalwani) [inline index.js:476]: Socket.io uses these, they will not conflict with route testing code. We are not sending peoples names with every socket message

File: index.js [hunk 2]
```diff
@@ -527,8 +527,8 @@ app.post('/event/unjoin', (req,res) => {
               'UNJOIN',
               result.rows[0],                                                       //All event info
               {
-                login: req.user.data.login,                                         //Login of who joined
-                name: `${req.user.data.firstname} ${req.user.data.lastname}`        //Name of who joined
+                login: req.body.login,                                         //Login of who joined
+                name: `${req.body.firstname} ${req.body.lastname}`        //Name of who joined
```
Comment (sanjayalwani) [inline index.js:531]: Likewise as before

**Review by jaskaran27177 — APPROVED** [review]

**Review by sanjayalwani — APPROVED** [review]

**Pull request merged**